pkg/sloop/webserver/webserver.go

```diff
@@ -213,10 +222,16 @@ func Run(config WebConfig, tables typed.Tables) error {
 	webFilesPath = config.WebFilesPath
 	server := &Server{}
 	server.mux = mux.NewRouter()
+	server.mux.Handle("/metrics", promhttp.HandlerFor(
```

Comment:
This is necessary to redirect http://localhost:8080/metrics to https://localhost:8080/{cluster}/metrics , so that it enables us to scrape from this endpoint into prometheus. If there is any other way to do it, do guide me will update.

Reply:
Yes but when we try accessing http://localhost:8080/metrics URL it is no accessible, only when we go to http://localhost:8080/ and then after it redirects we add "/metrics" after which we can view the metrics. So basically / gets rerouted to /cluster and then register paths works . Right? But for Prometheus to scrape from endpoint we need to absolute path which redirects to metrics directly without having us to add "/metrics" after going to http://localhost:8080/

Reply:
Why don't we have the path /cluster/metrics to Prometheus to scrape?

Reply:
May I know, how is this updated by spinnaker?
<img width="299" alt="Screenshot 2022-07-13 at 11 42 45 AM" src="https://user-images.githubusercontent.com/109100113/178664658-bbe6f971-34d6-4e45-907a-a91f6d77585c.png">
?

Reply:
@sana-jawad I have made the final commits. Please do review them.